src/components/LocalePicker.tsx

```diff
@@ -31,6 +35,7 @@ function LocalePicker({preferredLocale = CONST.LOCALES.DEFAULT, size = 'normal'}
         keyForList: language,
         isSelected: preferredLocale === language,
     }));
+    const shouldDisablePicker = AccountUtils.isValidateCodeFormSubmitting(account ?? {});
```

Comment:
I think it's more appropriate to use defaults in the prop destructuring.

Reply:
I don't fully understand that (warning: I am not very understanding of TS currently). Is TS not able to see that an `undefined` account would default to an empty object in the prop destructuring? If not, how about updating `isValidateCodeFormSubmitting` to accept an `OnyxEntry`?